workflow/rules/enrichment.smk

```diff
@@ -10,7 +10,9 @@ rule download_bioconductor_species_database:
     log:
         "logs/resources/bioconductor/{package}.log"
     shell:
-        "conda create --yes --quiet -p {params.path} --channel bioconda bioconductor-{wildcards.package}={params.version}"
+        "conda clean --all --yes --quiet; "
```

Comment:
Conda clean --all might destroy environments according to the docs. Although I have never seen this, we cannot enforce this for the end user. May I ask you to try this command before the workflow run? The snakemake-github-action has a `stagein` parameter for this purpose. Perhaps something is leaked in from the dockerfile.